src/util/XojMsgBox.cpp

```diff
@@ -9,7 +9,9 @@
 
 #ifdef _WIN32
 // Needed for help dialog workaround on Windows; see XojMsgBox::showHelp
-#include <shlwapi.h>
+#include <windows.h>
+//
```

Comment:
```suggestion
```

Reply:
windows.h must be included first.

Reply:
Then maybe add a comment saying just that instead of this empty comment line